### DISCUSSION TITLE
How to receive bytes above 7F ?

### DISCUSSION THREAD

**Discussion by rvalongo** [jchristn/SuperSimpleTcp#173] [Q&A]
The example is: Console.WriteLine($"[{e.IpPort}] {Encoding.UTF8.GetString(e.Data.Array, 0, e.Data.Count)}");

But what if I want to receive bytes form 0x80 to 0xFF ?

**Comment by jchristn** [comment]
Hi, you're free to use whatever encoding you wish. The data sent in the event is just byte data with no associated encoding or representation. You can use UTF16 if you wish.
